OloEditor/src/EditorLayer.cpp

```
@@ -1028,7 +1031,10 @@ namespace OloEngine
 
     void EditorLayer::OpenScene()
     {
-        std::string const filepath = FileDialogs::OpenFile("OloEditor Scene (*.olo)\0*.olo\0");
+        auto const dir = Project::GetActive()
+                             ? Project::GetAssetDirectory().string()
+                             : std::filesystem::current_path().string();
+        std::string const filepath = FileDialogs::OpenFile("OloEditor Scene (*.olo)\0*.olo\0", dir.c_str());
```

Clang (14.0.6)

[warning] variable 'filepath' is not initialized

(cppcoreguidelines-init-variables)